nservicebus/upgrades/5to6/distributor.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Deprecated Distributor in NServiceBus Version 6
+tags:
+ - upgrade
+ - migration
+related:
+ - samples/scaleout/distributor-upgrade
+---
+
+The [distributor](/nservicebus/scalability-and-ha/distributor) has been deprecated and replaced by [sender-side distribution](/nservicebus/msmq/scalability-and-ha/sender-side-distribution.md) with NServiceBus Version 6.
+
+## Client-side distribution
+
+With sender-side distribution, senders are aware of scaled out receivers, allowing senders to distribute messages directly to the receivers instead of using an intermediary distributor.
+
+
+## Upgrade scenario
+
+This process aims to allow upgrade in production environments without missing events and minimal downtime.
+
+WARNING: To avoid duplicate events, the Distributor node needs to be upgraded to a worker node using NServiceBus 6 (it can be decommissioned after all workers are upgraded to Version 6).
+
+* Upgrade all endpoints communicating to the Distributor (by commands or event subscription messages) to NServiceBus Version 6 first.
+* Shut down the Distributor instance.
+* [Upgrade the Distributor to NServiceBus Version 6](#upgrade-endpoint-to-version-6). This will migrate the Distributor endpoint to a regular worker instance.
```

Comment:
So, this assumes that it's a Master+Worker node, correct? If the user was running a Distributor only on a failover cluster, for instance, they probably wouldn't want to do this.